File: src/main/java/gestorAplicacion/comunicacion/ServicioFax.java
package gestorAplicacion.comunicacion;

/**
 * Implementación de un servicio de envío específico usando FAX.
 *
 * @author Juan Daniel Bula, Leidy Daniela Alzate, Oscar Andres Usuga
 */
public class ServicioFax extends ServicioEnvio {

    /**
     * Constructor asignando el mensaje al FAX.
     * @param mensaje mensaje a ser enviado.
     */
    public ServicioFax(String mensaje) {
        super(mensaje);
    }

    @Override
    public void enviar() {
        ServicioEnvio.mensajesEnviados.add("FAX: " + mensaje);
    }
}

File: python/src/gestorAplicacion/comunicacion/servicio_fax.py
from gestoraplicacion.comunicacion.servicio_envio import ServicioEnvio


class ServicioFax(ServicioEnvio):
    """
     Constructor asignando el mensaje al FAX.
     @param mensaje mensaje a ser enviado.
    """
    def __init__(self, mensaje):
        super().__init__(mensaje)

    def enviar(self):
        super()._mensajes_enviados.append("FAX: " + super()._mensaje)